client/starwhale/cli/assistance/wait_console.py

```diff
@@ -0,0 +1,41 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from starlette.middleware.cors import CORSMiddleware
+
+from starwhale.base.uri.resource import Resource, ResourceType
+
+
+class Command(BaseModel):
+    url: str
+    token: str
+
+
+def start() -> None:
+    app = FastAPI()
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=["*"],
+        allow_credentials=True,
+        allow_methods=["*"],
+        allow_headers=["*"],
+    )
+
+    @app.get("/alive")
+    async def alive() -> dict:
+        return {"message": "alive"}
+
+    @app.post("/resource")
+    async def resource(request: Command) -> dict:
+        rc = Resource(uri=request.url, token=request.token)
+        if rc.typ == ResourceType.model:
```

Comment:
the feature is import for dataset and runtime downloading, should we support them?

Reply:
I made this PR to check if the experimental feature is acceptable
OK, I'll add the support for the ds and rt in this PR

Reply:
Done